public/app/plugins/datasource/azuremonitor/dataquery.gen.ts

```diff
@@ -162,7 +162,11 @@ export const defaultAzureMetricQuery: Partial<AzureMetricQuery> = {
  */
 export interface AzureLogsQuery {
   /**
-   * If set to true the intersection of time ranges specified in the query and Grafana will be used. Otherwise the query time ranges will be used. Defaults to false
+   * If set to true the dashboard time range will be used as a filter for the query. Otherwise the query time ranges will be used. Defaults to false.
+   */
+  dashboardTime?: boolean;
+  /**
+   * @deprecated Use dashboardTime instead
```

Comment:
Do we need to mark this as deprecated or can we just remove it? We have a migration to remove it from existing queries, am I missing a reason to keep it around?